src/apps/sections/routers.py

```diff
@@ -0,0 +1,95 @@
+from fastapi import Depends, Request, Response, status
+from fastapi.routing import APIRouter
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from src.apps.sections.schemas import (
+    SectionInputSchema,
+    SectionOutputSchema,
+    SectionUpdateSchema,
+    SectionBaseOutputSchema
+)
+from src.apps.sections.services import (
+    create_section,
+    delete_single_section,
+    get_all_sections,
+    get_single_section,
+    update_single_section,
+)
+from src.apps.users.models import User
+from src.core.pagination.models import PageParams
+from src.core.pagination.schemas import PagedResponseSchema
+from src.core.permissions import check_if_staff
+from src.dependencies.get_db import get_db
+from src.dependencies.user import authenticate_user
+
+section_router = APIRouter(prefix="/sections", tags=["section"])
+
+
+@section_router.post(
+    "/",
+    response_model=SectionInputSchema,
+    status_code=status.HTTP_201_CREATED,
+)
+async def post_section(
+    section: SectionInputSchema,
+    session: AsyncSession = Depends(get_db),
+    request_user: User = Depends(authenticate_user),
+) -> SectionOutputSchema:
+    await check_if_staff(request_user)
+    return await create_section(session, section)
+
+
+@section_router.get(
+    "/",
+    response_model=PagedResponseSchema[SectionBaseOutputSchema],
+    status_code=status.HTTP_200_OK,
+)
+async def get_sections(
+    session: AsyncSession = Depends(get_db),
+    page_params: PageParams = Depends(),
+    request_user: User = Depends(authenticate_user),
+) -> PagedResponseSchema[SectionBaseOutputSchema]:
+    return await get_all_sections(session, page_params)
+
+
+@section_router.get(
+    "/{section_id}",
+    response_model=SectionOutputSchema,
+    status_code=status.HTTP_200_OK,
+)
+async def get_section(
+    section_id: str,
+    session: AsyncSession = Depends(get_db),
+    request_user: User = Depends(authenticate_user),
+) -> SectionOutputSchema:
+    await check_if_staff(request_user)
```

Comment:
i think that users should be able to see the single section, but with limited data
do sth similar to retirevieng single user:
example
if request_user.is_staff:
        return await get_single_user(session, user_id)
    return await get_single_user(session, user_id, output_schema=UserInfoOutputSchema)